### DISCUSSION TITLE
How to exclude files from being formatted by clang-format in a directory and all its subdirectories ?

### DISCUSSION THREAD

**Discussion by Pavi92** [microsoft/vscode-cpptools#12544] [Q&A]
Lets say I have the follwing project structure:

* Project
  * src
    * main.c
    * other.c
  * include
    * other.h
  * libs
    * lib1
      * lib1.c
      * lib1.h
    * lib2
      * lib2.c
      * lib2.h
   * .clang-format

How can I exclude all files inside the libs directory and subdirectories from being formatted ?
I tried to add a .clang-format-ignore file next to the .clang-format with follwing content:
`libs/*` 

But this does not work.

**Comment by sean-mcmanus** [comment]
That is tracked by https://github.com/microsoft/vscode-cpptools/issues/7543. You can upvote that.
